.gitignore

```diff
@@ -6,6 +6,12 @@
 # as most of those are useless.
 #==============================================================================#
 
+# Tool Runtime Dir
+.dotnet
```

Comment:
please move this to the end of the file with the rest of our dotnet-specific rules